src/classes/PageList.php

```diff
@@ -49,10 +54,23 @@ private function find_sibling_pages() {
   }
 
   private function find_child_pages() {
-    $args = array(
-      'parent' => $this->origin,
-      'sort_column' => 'menu_order',
-    );
+    //V: nick-branch
+    /**
+     * Might be missing ID
+     */
```

Comment:
Can you elaborate a little more about what you mean by "might be missing ID" please?

Reply:
Tried numerous ways of getting id from post but then all echoed back as nothing. get_the_ID() was the only way for me to a value

Reply:
I think it would make sense to store the current `$post` on the `PageList` object itself.  The post ID can be set by the constructor, rather than having to call out to WP for the post ID every time.